storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -16,7 +16,11 @@ To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage
 
 * *vSphere CSI Driver Operator*: The Operator provides a storage class, called `thin-csi`, that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
 
-* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs.
+* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs. In {product-title} 4.11, the driver version is 2.5.1.
```

Comment:
For versions, use {product-version} - for example, {product-title} {product-version}

Reply:
For this specific case, I chose to manually write it in so nothing slips. The driver is 1-to-1 with the OCP version and doesn't change, per dev.